contentpacks/khanacademy.py

```diff
@@ -356,11 +365,14 @@ def recurse_nodes(node, parent_path=""):
                             i += 1
 
         for child in children:
-            recurse_nodes(child, node.get("path"))
+            node_count += 1
```

Comment:
Let me see if I understand this algorithm correctly:

Say we have a `orange juice` with these parents:

```
seed > plant > orange > orange juice
        |
        |> dead plant > fertilizer
```

This algorithm does a depth-first traversal of the tree, which means that we have these `sort_order` values:

seed: 0
plant: 1
orange: 2
orange juice: 3

What would dead plant and fertilizer's sort order values be? It seems like they'll be 1 and 2, respectively, since the for loop wouldn't be aware of what the values in the recursive functions are.